### DISCUSSION TITLE
NVidia driver version

### DISCUSSION THREAD

**Discussion by Hezkore** [SpiralLinux/SpiralLinux-project#68] [Q&A]
How's the NVidia driver situation looking in Spiral Linux?
I'm running Debian atm. but upgrading my NVidia driver above version 470 has caused issues in the past.
Has Spiral Linux somehow made this step easier?

**Comment by geckolinux** [comment]
Hi there, thanks for the question. I would expect a similar experience to vanilla Debian. SpiralLinux does enable the non-free repository by default, and Synaptic is provided to make searching for and installing the correct Nvidia driver a bit easier. But the quality control and hardware integration as well as the kernel is all completely unmodified as Debian releases it. I would be very interested in knowing how the experience goes if you try SpiralLinux on your hardware, as I don't own any NVida test machines. Thanks!

**Comment by Vithos** [comment]
I'm struggling with this issue... I have 3080.  The only way to install nvidia-drivers is to disable the backports or I keep getting unmet dependencies... However, even then it doesn't install correctly. The nvidia X server setting isn't installed, but that's not the biggest issue.  It didn't install... I type 'sudo apt install nvidia-driver' and it says it's installed but when I type 'nvidia-smi' I get 'nvidia: command not found...'

Settings shows I'm still using llvmpipe

**Reply by geckolinux** [reply]
Hi there, sorry for the difficulty. Unfortunately I have no experience with Nvidia proprietary drivers. Please ask for help on https://www.reddit.com/r/debian , as SpiralLinux is built with unmodified Debian packages and uses upstream Debian package repos, so the behavior should be the same.

**Comment by CicadaSeventeen** [comment]
The only easy way to install Nvidia driver is to install versions from bullseye main.
see [here](https://github.com/SpiralLinux/SpiralLinux-project/issues/141)
